### ISSUE TITLE
About Page UI enhancement

### ISSUE THREAD

**Issue by IqraS-gif** [mansiruhil/fail-u-forward#242]
The current About page is not readable enough and is quite simple compared to the other components of the website. I can firstly improve the text color contrast and add dynamic CSS like glass morphism effect cards and animation to make it more engaging while making sure that the readability is maintained and proper info about the website is provided.

<img width="1438" height="867" alt="Image" src="https://github.com/user-attachments/assets/6448436c-9ac0-48ea-b62d-1a96e576fbf8" />

**Comment by Saga690** [comment]
unable to replicate 

<img width="1431" height="645" alt="Image" src="https://github.com/user-attachments/assets/801f7516-740f-400b-a4ea-7be5c2cd4a1a" />

**Comment by mansiruhil** [comment]
<img width="1366" height="644" alt="Image" src="https://github.com/user-attachments/assets/2fb6cf29-6542-49bb-9519-ccb646eaa0fa" />
It's already fine, no enhancement need
